includes/class-amp-theme-support.php

```diff
@@ -190,13 +205,16 @@ public static function init() {
 			}
 
 			add_action( 'widgets_init', [ __CLASS__, 'register_widgets' ] );
+			add_action( 'parse_query', [ __CLASS__, 'init_app_shell' ], 9 );
 
 			/*
 			 * Note that wp action is use instead of template_redirect because some themes/plugins output
 			 * the response at this action and then short-circuit with exit. So this is why the the preceding
```

Comment:
```suggestion
			 * Note that wp action is used instead of template_redirect because some themes/plugins output
			 * the response at this action and then short-circuit with exit. So this is why the preceding
```